addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/NeeoHandlerCallback.java

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Please review the packages layout for the binding. The non-exported packages should go into org.openhab.binding.neeo.internal.

Reply:
I've never understood this one since the manifest.mf defines what truely is exported - seems like having it at the package level is not only redundant, too wordy and breaks the 80/20 rules badly (less than 5% of the classes usually need exporting - why enforce this on the rest of the 95% of things).  I've gotten away from it in my last ones because it doesn't add anything and detracts from readability in my opinion.  I'll refactor if you feel strongly about it - but I really hate it.

Reply:
I think it is a part of the coding standard now and static code analyzer will not allow you to merge without following this rule.